src/cli/mod.rs

```diff
@@ -26,7 +26,7 @@ pub(crate) struct FlakeHubPushCli {
         default_value = "https://api.flakehub.com"
     )]
     pub(crate) host: url::Url,
-    #[clap(long, env = "FLAKEHUB_PUSH_VISIBLITY")]
+    #[clap(long, env = "FLAKEHUB_PUSH_VISIBILITY")]
```

Comment:
I think we probably need to accept the old version and the new version for a while, so we don't accidentally break users, OR accidentally publish something with the wrong visibility. Especially since the push action may be pinned, but the binary isn't.

Reply:
Okay, I added support for this, though it feels fairly inelegant. Let me know if a better implementation comes to mind.

Reply:
cc @cole-h for review